ansible/roles/gateone-gen-sshkey/tasks/main.yml

```diff
@@ -30,5 +29,30 @@
   delegate_to: gateone
 
 - name: finally, transfer the public key locally
-  fetch: src="/var/lib/gateone/users/{{ ATMOUSERNAME }}/.ssh/{{ ATMOUSERNAME }}_default.pub" dest="{{ GATEONE_LOCAL_PUBKEY_DIR | default('/tmp/') }}" flat=yes
+  fetch: src= "/var/lib/gateone/users/{{ ATMOUSERNAME }}/.ssh/{{ ATMOUSERNAME }}_default.pub" dest= "{{ GATEONE_LOCAL_PUBKEY_DIR | default('/tmp/') }}" flat=yes
   delegate_to: gateone
+
+# THESE values were in atmo-user-ssh-key and throw an error at thi stage.
+# TODO: discuss if this is required..
+
+# - name: grab local copy of user ssh private key
+#   set_fact: pk_file= "{{ lookup('file', '/tmp/' + ATMOUSERNAME + '_default.pub') }}"
+#   tags: debug
+# 
+# - name: distribute the generated gate one key to the instance
+#   authorized_key: >
+#     key= '{{ pk_file }}'
+#     state=present
+#     user= "{{ item }}"
+#   when: pk_file is defined
+#   with_items:
+#     - "{{ ATMOUSERNAME }}"
+#   tags:
+#     - ssh-keys
+#     - debug
+# 
+# - name: remove user key from controller node
+#   local_action: file path=/tmp/{{ ATMOUSERNAME }}_default.pub state=absent
+#   tags:
+#     - ssh-keys
+#     - delete-key
```

Comment:
As noted [above on L35](https://github.com/iPlantCollaborativeOpenSource/atmosphere-ansible/pull/25/files#diff-8a0ac20f8fd8e19efb9f578e63c14697R35) I commented it out because I was receiving this error:
```
fatal: [129.114.5.19]: FAILED! => {"failed": true, "msg": "the file_name '/tmp/sgregory_default.pub' does not exist, or is not readable"}
```